protocol-designer/src/NavigationBar.tsx

```diff
@@ -0,0 +1,109 @@
+import { actions as loadFileActions } from './load-file'
+import type { ThunkDispatch, RouteProps } from './types'
+
+export function NavigationBar({
+  routes,
+}: {
+  routes: RouteProps[]
+}): JSX.Element {
+  const { t } = useTranslation('shared')
+  const navRoutes = routes.filter(
+    (route: RouteProps) => route.navLinkTo !== '/createNew'
+  )
+  const location = useLocation()
+  const dispatch: ThunkDispatch<any> = useDispatch()
+  const navigate = useNavigate()
+  const loadFile = (
+    fileChangeEvent: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    if (window.confirm(t('confirm_import') as string)) {
+      dispatch(loadFileActions.loadProtocolFile(fileChangeEvent))
+      navigate('/overview')
+    }
+  }
+  const filteredNavPaths =
+    location.pathname === '/createNew' || location.pathname === '/'
+
+  return (
+    <Flex flexDirection={DIRECTION_COLUMN}>
+      <Flex
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
+        padding={`${SPACING.spacing12} ${SPACING.spacing40}`}
+      >
+        <Flex gridGap={SPACING.spacing8} alignItems="center">
+          <StyledText desktopStyle="bodyLargeSemiBold">
+            {t('opentrons')}
+          </StyledText>
+          <StyledText desktopStyle="bodyLargeSemiBold" color={COLORS.purple50}>
+            {t('protocol_designer')}
+          </StyledText>
+          <StyledText desktopStyle="captionRegular" color={COLORS.grey50}>
+            {t('version', { version: process.env.OT_PD_VERSION })}
+          </StyledText>
+        </Flex>
+        <Flex gridGap={SPACING.spacing40} alignItems="center">
+          {location.pathname === '/createNew' ? null : (
+            <NavbarLink key="createNew" to={'/createNew'}>
+              <StyledText desktopStyle="bodyDefaultRegular">
+                {t('create_new_protocol')}
+              </StyledText>
+            </NavbarLink>
+          )}
+          <StyledLabel>
+            <StyledText desktopStyle="bodyLargeRegular" color={COLORS.grey60}>
+              {t('import')}
+            </StyledText>
+            <input type="file" onChange={loadFile}></input>
+          </StyledLabel>
+        </Flex>
+      </Flex>
+      {/* TODO(ja, 8/12/24: delete later. Leaving access to other
+      routes at all times until we make breadcrumbs and protocol overview pg */}
+      {filteredNavPaths ? null : (
+        <Flex
+          backgroundColor={COLORS.blue20}
+          padding={`${SPACING.spacing12} ${SPACING.spacing40}`}
+          gridGap={SPACING.spacing40}
+        >
+          TODO add breadcrumbs here
+          {navRoutes.map(({ name, navLinkTo }: RouteProps) => (
+            <NavbarLink key={name} to={navLinkTo}>
+              <StyledText desktopStyle="bodyDefaultRegular">{name}</StyledText>
+            </NavbarLink>
+          ))}
+        </Flex>
+      )}
+    </Flex>
+  )
+}
+
+const NavbarLink = styled(NavLink)`
+  color: ${COLORS.black90};
+  text-decoration: none;
+  align-self: ${ALIGN_STRETCH};
```

Comment:
![Screenshot 2024-08-13 at 8 02 49 AM](https://github.com/user-attachments/assets/d2a73b6d-52bb-4c12-8140-5227935e83cb)


```suggestion
  align-self: ${ALIGN_CENTER};
```

or (if need to keep `stretch`

```ts
<Flex alignItems={ALIGN_CENTER}>
              <NavbarLink key="createNew" to={'/createNew'}>
                <StyledText desktopStyle="bodyDefaultRegular">
                  {t('create_new_protocol')}
                </StyledText>
              </NavbarLink>
            </Flex>
```

Reply:
thanks! i couldn't figure out what was happening, didn't see this align-self i added